rest/rest-resources/src/main/java/org/apache/brooklyn/rest/transform/EntityTransformer.java

```diff
@@ -162,7 +163,7 @@ public static EntityConfigSummary entityConfigSummary(Entity entity, ConfigKey<?
             SensorTransformer.addNamedAction(lb, na, entity.getConfig(config), config, entity);
         }
     
-        return entityConfigSummary(config, label, priority, lb.build());
+        return entityConfigSummary(config, label, priority, false, lb.build());
```

Comment:
Well, yes and no. I mean, the UI does not check a `pinned` flag for managed entities but only for catalog templates. So technically, config will never be invisible. 
However, I do agree that the REST API shouldn't return a `pinned: false` for this case for I'll update for a nullable field.